components/auth/ReVerify.tsx

```diff
@@ -84,7 +92,9 @@ export const ReVerify = ({
           ) : undefined}
         </Alert>
       )}
-      <h1 className="border-0 mt-6 mb-6">{t("reVerify.title")}</h1>
+      <h1 ref={headingRef} className="border-0 mt-6 mb-6">
```

Comment:
I completely forgot to add this. So when a sort of SPA model is being used to update "pages" the user should be notified in some way that the page has changed. This is accomplished by updating the HEAD TITLE el above but to make it really clear, focussing the main heading, really helps. Both are announced actually, this really gives clarity and also takes care of managing the user focus.
This could also be accomplished in other ways like wrapping the entire "page" in a live-region. The problem with this is that Assistive Technologies will often ignore this update because the diff in the accessibility tree (tree built from DOM with all the accessible info) is too big / lazy AT programmers? :).